packages/frontend-language-toggle/src/template.test.ts

```diff
@@ -58,7 +58,7 @@ describe("languageSelect Component", () => {
 
   describe("renders active language as a span, and inactive language as a link", () => {
     it("displays cy active language as a span, and inactive language as a link", () => {
-      const mockParams = {
+      mockParams = {
```

Comment:
Can’t we just spread the object and update only the specific property we need?

Reply:
Yes we can, I've updated accordingly